src/common/page404/Page404.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ErrorPage from '../../profile/components/errorPage/ErrorPage';
+
+function Page404(): React.ReactElement {
```

Comment:
I wouldn't use the text "404" in this component's name, because this component **does not** make the server return a 404 HTTP status code, when a not found page (or route) is requested. Perhaps call this component `PageNotFound` or something like that.

Reply:
"404Page" was first my thought, but a function name cannot start with a number. Sounds weird that naming it "Page404" would suggest that a React component returns http status 404. 

Renamed it to PageNotFound.

Reply:
404 is just a number, without any inherent meaning. If interpreted as an HTTP status code, it means "Not Found". In other contexts it can mean something else or be meaningless. The feature in this PR has nothing to do with HTTP or HTTP status codes. So in this context the number 404, at least to me, means nothing (or then I can't figure out what the context here would be where 404 has some meaning).

The 404 is not in code any more, but is still in the commit message. Could remove it from there too.